tests/js/client/shell/shell-vst-dump.js

```diff
@@ -0,0 +1,67 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global arango, assertEqual */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief ArangoTransaction sTests
+// /
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2018 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is triAGENS GmbH, Cologne, Germany
```

Comment:
Is this coming from older code? Should it not read ```Copyright holder is ArangoDB GmbH, Cologne, Germany```?

Reply:
Yes, the skeleton for that test file came from another, existing test.
I adjusted the comments section in my PR now, but I think we should clean up all the other tests in `tests/js` once in a different PR.